features/user_can_upload_image.feature

```diff
@@ -0,0 +1,21 @@
+Feature: User can upload image
+
+    As a user
```

Comment:
Update user story to reflect what you guys actually implemented (user can add profile picture)